cranelift/codegen/src/isa/x64/inst/args.rs

```diff
@@ -295,9 +295,13 @@ impl SseOpcode {
     pub(crate) fn to_string(&self) -> String {
         match self {
             SseOpcode::Addss => "addss".to_string(),
+            SseOpcode::Divss => "divss".to_string(),
             SseOpcode::Subss => "subss".to_string(),
+            SseOpcode::Maxss => "maxss".to_string(),
+            SseOpcode::Minss => "minss".to_string(),
             SseOpcode::Movss => "movss".to_string(),
             SseOpcode::Movsd => "movsd".to_string(),
+            SseOpcode::Mulss => "mulss".to_string(),
```

Comment:
I just realized we could in theory just suppress this `to_string` function, since the `SseOpcode` struct implements `ToString` (with all the existing variants implemented).

Reply:
@bnjbvr .. By suppress do you mean to just delete this function with this patch? I did comment it out and so no issues in testing or running a simple example. Not sure if the lowercase could come into place somewhere, but I can delete this since it doesn't appear to be necessary.

Reply:
Yes, that's what i meant, sorry for using unclear vocabulary. To wit: the `ToString` trait implementation adds the `to_string` function, maintaining feature completeness here.